.expeditor/scripts/build_mac_hab_binary.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/local/bin/bash
+
+set -euo pipefail
+
+source .expeditor/scripts/shared_release_habitat.sh
+
+# Get secrets! (our auth token and aws creds should be auto-injected but there's a bug:
+# https://github.com/chef/ci-studio-common/issues/200)
+eval "$(vault-util fetch-secret-env)"
+
+export HAB_AUTH_TOKEN="${ACCEPTANCE_HAB_AUTH_TOKEN}"
+export HAB_BLDR_URL="${ACCEPTANCE_HAB_BLDR_URL}"
+
+channel=$(get_release_channel)
+
+echo "--- Channel: $channel - bldr url: $HAB_BLDR_URL"
+
+echo "--- Installing buildkite agent"
+brew tap buildkite/buildkite
+brew install --token="$BUILDKITE_AGENT_ACCESS_TOKEN" buildkite-agent
+
+echo "--- Installing mac bootstrap package"
+# TODO: subscribe to releases: https://github.com/habitat-sh/release-engineering/issues/84
```

Comment:
I think we agreed to use `TODO` to mark things that should happen _before_ merge (such as create the issue for future fork). Since that issue is already created and linked here, I think we can remove the `TODO` and just leave the issue link.

Reply:
That seems counterintuitive to me but if that's what consensus is I'm fine with it!

Reply:
The idea is that `TODO`s in code (rather than as issues) make it hard to keep track of the work we want to do in the future because it leads to fragmentation. We want all our planned work to exist in issues to capture context and be discoverable. It's fine to link issues in code, but we're trying to get away from merging the text `TODO` specifically, because we want it to serve a different purpose.

The idea there is that putting `TODO` in a bit of code is a fine practice to keep your momentum so you don't have to to search for or create the issue right away and can keep working on the code task at hand (perhaps the TODO gets resolved without needing an issue at all). Then later, when things are ready to merge, we can go back and make sure all the `TODO` get dealt with appropriately (either by code changes or issue creation) _before_ merge.

In this example, while working on the code, writing
```bash
# TODO: subscribe to releases
```
and then before merging, creating the issue changing that `TODO` line to:
```bash
# subscribe to releases: https://github.com/habitat-sh/release-engineering/issues/84
```
is the expected workflow. `TODO` is just a marker that can help us remember to do things before merge without getting to distracted. Does that make sense?

Reply:
I understand. That's not a workflow I've used, but like I said if that's how we're doing it, I'm fine with it! :) I feel like having a marker in the code to link to an issue (todo, fixme, whatever) is easier to search for unless you're linking directly in the issue to the code in question, which may be what y'all are doing already.

Reply:
If the issue is searchability, grepping for `"issues/\d+"` works pretty well